### ISSUE TITLE
[request] windows support

### ISSUE THREAD

**Issue by Ant1r** [pd-externals/command#18]
obviously...

**Comment by reduzent** [comment]
Agreed. I don't recall exactly, but [command] uses a few UNIX-isms (execvp?) that I don't know yet how to translate to Windows. And I must say, I'm totally not a well-seasoned C programmer. 

If you feel like submitting a PR, that would much appreciated, of course.
